roles/openshift_prometheus/tasks/oc_apply.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Checking generation of {{file_content.kind}} {{file_content.metadata.name}}
```

Comment:
I also found `oc_route` module, but I noticed that the existing modules don't have `annotations` parameter, which I can currently set with a template. 
Is this planned? Can I add this ability or is someone already on it? is there a different way to add annotations? I need this on `oc_service` and `oc_serviceaccount` modules
@kwoodson @ewolinetz